packages/airnode-deployer/src/handlers/aws/index.ts

```diff
@@ -61,5 +61,6 @@ export async function testApi(event: any) {
     return { statusCode: 400, body: JSON.stringify({ error: err.toString() }) };
   }
 
-  return { statusCode: 200, body: JSON.stringify(result) };
+  // NOTE: We do not want the user to see {"value": <actual_value>}, but the actual value itself
+  return { statusCode: 200, body: result!.value };
```

Comment:
I've remove `toString` because the output was unusable. Now it's proper JSON and that makes things like formatting a bit easier.

Reply:
https://github.com/api3dao/api3-docs/pull/417